### DISCUSSION TITLE
Settings for autoplay

### DISCUSSION THREAD

**Discussion by ramblan** [Valexr/Slidy#12] [General]
I am searching to find an image carousel in Svelte, and this one seems to be really good. I have trouble to find an autoplay function. Is it implemented and in that case what settings are there? /Tommy

**Comment by Valexr** [comment]
This function not implemented in core, but you can make it simply externally – [examaple](https://svelte.dev/repl/c3f50a4277384ebf860fcb1bb8d26dae)

**Comment by yhdgms1** [comment]
Today there is support for the autoplay function in Slidy! Look for the [autoplay plugin](https://github.com/Valexr/Slidy/blob/main/packages/plugins/src/lib/autoplay/README.md)
